Winapp2.ini

```diff
@@ -5734,12 +5747,33 @@ Default=False
 FileKey1=%LocalAppData%\CrypTool2\Temp|*.*|REMOVESELF
 RegKey1=HKCU\Software\CrypTool2.0|recentFileList
 
-[Crysis 3 *]
+[Crysis *]
 Section=Games
-Detect=HKLM\Software\Crytek\Crysis 3
+Detect1=HKLM\Software\Valve\Steam\Apps\17300
+Detect2=HKLM\Software\Valve\Steam\Apps\17330
+Detect3=HKLM\Software\Valve\Steam\Apps\108800
+Detect4=HKLM\Software\WOW6432Node\Origin Games\71503
+Detect5=HKLM\Software\WOW6432Node\Origin Games\1003295
+Detect6=HKLM\Software\WOW6432Node\Origin Games\1003302
+Detect7=HKLM\Software\WOW6432Node\Origin Games\1003310

```

Comment:
Detect does not require "WOW6432Node". (They were all removed during the overhaul.)